pennylane/pauli/dla/center.py

```diff
@@ -16,9 +16,50 @@
 from typing import Union
 
 import numpy as np
+from scipy.linalg import norm, null_space
 
 from pennylane.operation import Operator
 from pennylane.pauli import PauliSentence, PauliWord
+from pennylane.pauli.dla import structure_constants
+
+
+def _intersect_bases(basis_0, basis_1):
+    r"""Compute the intersection of two vector spaces that are given by a basis each.
+    This is done by constructing a matrix [basis_0 | -basis_1] and computing its null space
+    in form of vectors (u, v)^T, which is equivalent to solving the equation
+    ``basis_0 @ u = basis_1 @ v``.
```

Comment:
Can you add these inside inline code blocks or inline latex blocks?

Reply:
They are currently inline, aren't they? :thinking: Do you mean code/math blocks? :)